browser/ephemeral_storage/ephemeral_storage_tab_helper.cc

```diff
@@ -124,10 +129,31 @@ void EphemeralStorageTabHelper::CreateEphemeralStorageAreasForDomainAndURL(
                 kSessionStorageSuffix))
           : base::nullopt);
 
+  if (base::FeatureList::IsEnabled(
+          net::features::kBraveEphemeralStorageKeepAlive)) {
+    // keep the ephemeral storage alive for some time to handle redirects
+    // including meta refresh or other page driven "redirects" that end up back
+    // at the original origin
+    base::SequencedTaskRunnerHandle::Get()->PostDelayedTask(
+        FROM_HERE,
+        base::BindOnce([](scoped_refptr<content::TLDEphemeralLifetime>
+                              tld_ephemeral_lifetime) {},
+                       tld_ephemeral_lifetime_),
+        g_storage_keep_alive_for_testing.is_min()
```

Comment:
also, maybe a nit, but why use ```is_min``` and not ```is_zero```? min is a negative value, not a default one. ```TimeDelta``` has a default constructor that sets an internal value to 0, and it can be checked with ```is_zero``` which is clearer imo.

Reply:
because 0 might be a value we want to use in testing when the feature flag goes away

Reply:
`is_min` seemed  more clear as an invalid value vs something that would be set in `SetKeepAliveTimeDelayForTesting`